src/Discovery/Discover.cs

```diff
@@ -74,6 +74,60 @@ public bool BeginDiscovery()
             if (UserInputObj.CancellationContext.IsCancellationRequested)
                 UtilityFunctions.InitiateCancellation(UserInputObj);
 
+            int excelCreationPercentProgress = 0;
+            if (UserInputObj.IsImportDiscovery)
+                excelCreationPercentProgress = PerformImportDiscovery(masterSitesObj);

```

Comment:
This check can be modified to "userInputObj.ApplianceSources" cotaining "import"